DataLayer/LightsData/LightsData.cpp

```diff
@@ -11,7 +11,7 @@ namespace
 }
 
 LightsData::LightsData()
-    : lightStatus_(0)
+    : alive_(false), lightStatus_(0)
```

Comment:
For inline construction, let's keep it so it's split like 
`: alive_(false)`
`, lightStatus_(0)`